### ISSUE TITLE
Another implementation

### ISSUE THREAD

**Issue by flyingP0tat0** [jeroenvervaeke/redux-rs#4]
I created a Redux implementation in Rust as well. It is currently under [flyingP0tat0/redux-rs](https://github.com/flyingP0tat0/redux-rs).
Now, I realized that you have already implemented parts of Redux in this crate (because the name `redux-rs` was already used). At this point, I don't intend to just put up a second Redux implementation on crates.io, rather I would like to somehow contribute to what has already been done. The thing is that my version includes things like documentation, benchmarks and middleware. But on the other hand it is more or less a re-write of the code in this repository, so making a pull request would result in completely changed code.
So if you would accept somehow merging the two projects, that would be great. One could, for example, create a new GitHub organisation for the project.
After all, it's just about not having two so identical projects.

Let me know what you think.

**Comment by jeroenvervaeke** [comment]
I've taken a look at your project and it looks indeed very similar to my project, except that your project is a lot more complete than mine. I've thought about updating it this week, so your issue was issued just in time :)

I think it's a great idea to create a new Github organization for the project. That way we can also create middleware for the library and host it on there.

I did create the organization and I've invited you (to both the new organization and crates.io)
